RpcNep5Balances/RpcNep5Balances.cs

```diff
@@ -0,0 +1,259 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using Encoding = System.Text.Encoding;
+using Microsoft.AspNetCore.Http;
+using Neo.IO.Caching;
+using Neo.IO.Data.LevelDB;
+using Neo.IO.Json;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.Persistence.LevelDB;
+using Snapshot = Neo.Persistence.Snapshot;
+using Neo.SmartContract;
+using Neo.Network.RPC;
+using Neo.VM;
+using Neo.Wallets;
+
+namespace Neo.Plugins
+{
+    public class RpcNep5Balances : Plugin, IPersistencePlugin, IRpcPlugin
```

Comment:
Originally when I started writing the plugin it only handles balances, since it can handle both balance and transfer history, I’ll rename the plugin to `RpcNep5Plugin`

Reply:
Decided to rename to `RpcNep5Tracker`.